wire-schema/src/test/java/com/squareup/wire/schema/internal/parser/ProtoParserTest.kt

```diff
@@ -0,0 +1,2341 @@
+                .types(
+                        ImmutableList.of(
+                                MessageElement.builder(location.at(1, 1))
+                                        .name("Chickens")
+                                        .fields(
+                                                ImmutableList.of(
+                                                        FieldElement.builder(location.at(2, 3))
+                                                                .label(OPTIONAL)
+                                                                .type("bool")
+                                                                .name("koka_ko_koka_ko")
+                                                                .tag(1)
+                                                                .options(
+                                                                        ImmutableList.of(
+                                                                                OptionElement.create("old_default", Kind.BOOLEAN, "true")
+                                                                        )
+                                                                )
+                                                                .build(),
+                                                        FieldElement.builder(location.at(3, 3))
+                                                                .label(OPTIONAL)
+                                                                .type("bool")
+                                                                .name("coodle_doodle_do")
+                                                                .tag(2)
+                                                                .options(
+                                                                        ImmutableList.of(
+                                                                                OptionElement.create("delay", Kind.NUMBER, "100", true),
+                                                                                OptionElement.create("old_default", Kind.BOOLEAN, "false")
+                                                                        )
+                                                                )
+                                                                .build(),
+                                                        FieldElement.builder(location.at(4, 3))
+                                                                .label(OPTIONAL)
+                                                                .type("bool")
+                                                                .name("coo_coo_ca_cha")
+                                                                .tag(3)
+                                                                .options(
+                                                                        ImmutableList.of(
+                                                                   
```

Comment:
nit: `listOf`

Reply:
i'll fix that when converting the models themselves.